src/frontend/get-github-setup.ts

```diff
@@ -14,17 +14,39 @@ export default async (req: Request, res: Response): Promise<void> => {
 	req.log.info("Received get github setup page request");
 	const { jiraHost } = res.locals;
 
+	const installationId = req.originalUrl.split("=")[1].split("&")[0]
 	let redirectUrl = getJiraMarketplaceUrl(jiraHost);
 
+	const {	github, client } = res.locals;
+
+	const { data: { installations } } = await github.apps.listInstallationsForAuthenticatedUser();
+	const { data: info } = await client.apps.getAuthenticated();
+
+	const installation = installations.filter((item) => item.id === Number(installationId));
```

Comment:
1. Can we get the exact installation ID from GitHub rather than filtering them manually?

2. Should we use new GitHub Client instead of Octokit? (Maybe not now, because it might require some middleware changes)

Reply:
- [ ] Query the exact installationId from github

Reply:
1. Not that I could find. I was hoping GH had an API that I could call with the installationId but there doesn't appear to be one. Getting all the orgs where the app was installed appears to be the only option.

2. Yes, yes we should 😄 Will update.

Reply:
Interesting, is it possible to get the org ID from our DB somehow and then query by Org?

Reply:
Not at this point in the flow. The installations table doesn't have this data and, at this point, no subscription has been made (happens after connect is clicked).

Reply:
https://github.com/atlassian/github-for-jira/pull/1088#discussion_r808646896 I don't think there is a way to do this.

Reply:
Yeah, installationId is not a first class citizen apparently. I can't find any REST or GraphQL APIs that we can use to get it.  kinda stupid really.